crates/bevy_ecs/src/observer/mod.rs

```diff
@@ -114,6 +148,15 @@ pub struct ObserverTrigger {
     pub entity: Entity,
 }
 
+/// Determines if the event should propagate to other entities.
+#[derive(Eq, PartialEq, Copy, Clone, Debug)]
+pub enum Propagation {
```

Comment:
Is this enum necessary? I feel like this would be better as a `bool` because most of the time the thing that users will use to determine propagation is a boolean expression.

Reply:
I agree. I think making this a bool, as a field that you directly access improves the legibility / usability of this (and removes the need for 3 separate accessor methods).

I'm putting together the change now for consideration.

Reply:
Actually I'll take back that it should be a public field. In theory that would make it possible to override mutable borrow with a different one (although in practice this would be very challenging because it would need the same `w` lifetime). Better to keep that door closed though.

Reply:
I've swapped this for a bool.